package.json

```diff
@@ -4,8 +4,8 @@
   "main": "node_modules/expo/AppEntry.js",
   "scripts": {
     "start": "expo start",
-    "android": "expo start --android",
```

Comment:
This changes the behaviour of the app in expo go, is it on purpose ? Here is what chatGPT has told me on the difference between start and run :

    Development Environment vs. Native Environment:
        expo start: Runs the app within the Expo Go app, which contains pre-built native code and APIs provided by Expo.
        expo run: Builds and runs the app with your specific configurations and native code, directly on the emulator, simulator, or device.

    Quick Iteration vs. Detailed Testing:
        expo start is optimal for developers looking to quickly make changes and test outcomes in a dynamic development environment.
        expo run is crucial for testing more complex integrations and performance issues that can only be identified in a standalone app environment.

    Managed vs. Bare Workflow Features:
        While both commands can be used in a managed workflow with Expo, expo run steps closer to what is traditionally considered a bare workflow by directly interacting with the native codebase.